database/info.txt

```diff
@@ -1 +0,0 @@
-This folder contains the data
```

Comment:
Please revert this change, this file is used to ensure the `database` directory is present when cloning the Git repo.